components/core/src/clp/clo/ResultsCacheClient.hpp

```diff
@@ -0,0 +1,61 @@
+#ifndef CLP_CLO_MONGODBCLIENT_HPP
+#define CLP_CLO_MONGODBCLIENT_HPP
+
+#include <string>
+
+#include <mongocxx/client.hpp>
+#include <mongocxx/collection.hpp>
+#include <mongocxx/exception/exception.hpp>
+#include <mongocxx/uri.hpp>
+
+#include "../Defs.h"
+#include "../TraceableException.hpp"
+
+namespace clp::clo {
+/**
+ * Class encapsulating a MongoDB client used to send query results to results cache.
```

Comment:
```suggestion
 * Class encapsulating a MongoDB client used to send query results to results the cache.
```

Reply:
What does "results the cache" mean?

Reply:
It means I'm tired, lol. "the results cache."